packages/rn-tester/RNTesterIntegrationTests/RCTLoggingTests.m

```diff
@@ -114,7 +114,7 @@ - (void)testLogging
 
   XCTAssertEqual(_lastLogLevel, RCTLogLevelError);
   XCTAssertEqual(_lastLogSource, RCTLogSourceJavaScript);
-  XCTAssertEqualObjects(_lastLogMessage, @"Error: Throwing an error");
+  XCTAssertTrue([_lastLogMessage containsString:@"Error: Throwing an error"]);
```

Comment:
On that note, shall we have two targets for integration tests and two targets for unit tests and run them on both Hermes and JSC? To be honest, it sounds to me like an overkill, but would like to discuss this idea.

Reply:
I wonder if we can achieve that simply by passing something like `GCC_PREPROCESSOR_DEFINITIONS ="RCT_USE_HERMES=0"` to the `xcodebuild` command, rather than adding extra targets.

Either way, it might be better to do that in a follow-up PR?

Reply:
Yeah, we can play around with it later. Mainly wanted to decide on a conceptual level what is it that we want to test - it seems reasonable to have a separate CircleCI job to test both JSC and Hermes at the same time.

In such case, we would have 3 out of 4 tests passing - dynamic frameworks and Hermes combination would fail.